examples/slack/README.md

```diff
@@ -0,0 +1,51 @@
+# Slack Example
+
+This example shows how to send cost estimates to Slack by combining the Infracost actions with the official [slackapi/slack-github-action](https://github.com/slackapi/slack-github-action) repo. 
+
+Slack message blocks have a 3000 char limit so the Infracost CLI automatically truncates the middle of `slack-message` output formats.
+
+For simplicity, this is based off the terraform-plan-json example, which does not require Terraform to be installed.
+
+<img src=".github/assets/slack-message.png" alt="Example screenshot" />
+
+[//]: <> (BEGIN EXAMPLE)
+```yml
+name: Slack
+on: [pull_request]
+
+jobs:
+  slack:
+    name: Slack
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      
+      - name: Setup Infracost
+        uses: infracost/actions/setup@v1
+        with:
+          api-key: ${{ secrets.INFRACOST_API_KEY }}
+
+      - name: Generate Infracost JSON
+        id: infracost-commands
+        run: |
+          infracost breakdown --path=examples/thresholds/code/plan.json --format json --out-file /tmp/infracost.json
+          echo "::set-output name=slack-message::$(infracost output --path /tmp/infracost.json --format slack-message --show-skipped)"
```

Comment:
It might be clearer if we put this in a separate step called "Generate Slack message"